appveyor.yml

```diff
@@ -1,6 +1,16 @@
+clone_folder: c:\go\src\github.com\github\git-lfs
+
+environment:
+  GOPATH: c:\go
+
 install:
+  - set PATH=%GOPATH%\bin;%PATH%;
+  - go version
```

Comment:
Even if you left this in for debugging, I don't think it adds much value as we have the standard `Using go1.6.2` lines anyway.